### PR TITLE
Gonzalo reporting for duty

### PR DESCRIPTION
**Pull request by gonmendoza**

### CODE REVIEW

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Hi @gonmendoza! The change is fine, however if you noticed, the commits are not properly attributed to you:

https://github.com/Juanlu001/ie-mbd-advanced-python/pull/13/commits

The reason is likely that the email address is not properly configured:

https://github.com/gonmendoza/ie-mbd-advanced-python/commit/4b579ae4de5baabfdb883fc9124dc4d600afdeb8.patch

The commit is attributed to your gmail address, but perhaps you registered in GitHub with your IE address? Review your email addresses here:

https://github.com/settings/emails

Please let me know if you have questions.

**Comment by gonmendoza** [comment]
> 
> 
> Hi @gonmendoza! The change is fine, however if you noticed, the commits are not properly attributed to you:
> 
> https://github.com/Juanlu001/ie-mbd-advanced-python/pull/13/commits
> 
> The reason is likely that the email address is not properly configured:
> 
> https://github.com/gonmendoza/ie-mbd-advanced-python/commit/4b579ae4de5baabfdb883fc9124dc4d600afdeb8.patch
> 
> The commit is attributed to your gmail address, but perhaps you registered in GitHub with your IE address? Review your email addresses here:
> 
> https://github.com/settings/emails
> 
> Please let me know if you have questions.

Im trying again, this time putting the file in the correct folder.

The email is the one I created the github account with so im not sure why im getting that issue

**Review by gonmendoza — COMMENTED** [review]
i was missing the 1990 from the email

**Comment by astrojuanlu** [comment]
Right! Your last commit is properly attributed to <gonmendoza1990@gmail.com>, but the previous ones are to <gonmendoza@gmail.com>. Perhaps that's the problem?

**Comment by astrojuanlu** [comment]
:+1: we wrote at the same time

File: Gonzalo.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Gon mola mogollon
```
Comment (astrojuanlu) [inline Gonzalo.txt]: Can you please remove this file?

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Almost there! We have to remove one of the files

**Review by astrojuanlu — APPROVED** [review]
:rocket:

**Comment by astrojuanlu** [comment]
:warning: Next steps :warning: 

Your pull request has been merged to `master`. That means that, if you don't do anything, your local `master` and `gonmendoza/master` are outdated with respect to `origin/master`. On the other hand, you should avoid working on this branch anymore in the future.

To put yourself in a clean state again, you have to:

1. Click "remove branch" here in this pull request (don't click "remove fork"!)
2. `git checkout master`
3. `git fetch origin` (assuming https://github.com/Juanlu001/ie-mbd-advanced-python/ is `origin` for you)
4. `git merge --ff-only origin master` (this will update your local `master` with `origin/master`, and fail if you accidentally made any commit in `master`)
5. `git fetch -p gonmendoza` (assuming https://github.com/gonmendoza/ie-mbd-advanced-python/ is `gonmendoza` for you) :star2: this will acknowledge the removal of the branch :star2:
6. `git push gonmendoza master` (this will update your fork with respect to `origin`)

The objective of this process, **which has to be repeated for every pull request**, is to be able to submit a second one without git problems.

~ If in doubt, please ask ~

:rainbow:

**Pull request merged**